discord/state.py

```diff
@@ -780,6 +780,10 @@ def parse_guild_members_chunk(self, data):
         log.info('Processed a chunk for %s members in guild ID %s.', len(members), guild_id)
         self.process_listeners(ListenerType.chunk, guild, len(members))
 
+    def parse_webhooks_update(self, data):
+        channel = self.get_channel(int(data["channel_id"]))
```

Comment:
Two things:

1. I prefer single quotes over double quotes
2. Are you sure this can't be `None`? I tend to have an internal helper for those cases.